src/autorest-core/resources/default-configuration.md

```diff
@@ -27,18 +27,339 @@ plugins:
   swagger-loader: 42
 ```
 
-## Output Artifacts
+### Graph
 
-What to write out.
+#### Reflection
 
 ``` yaml
+pipeline:
+  pipeline-emitter: # emits the pipeline graph
+    scope: scope-pipeline-emitter
+```
+
+#### Loading
+
+``` yaml
+pipeline:
+  swagger-document/loader:
```

Comment:
abstract representation of the pipeline. contains some duplication, will make that more and more convenient (goes hand in hand with making pipeline customization easier)